go/pserver/cclient/cclient.go

```diff
@@ -123,9 +123,8 @@ func paddle_begin_init_params(client C.paddle_pserver_client) C.int {
 func paddle_init_param(client C.paddle_pserver_client, param C.paddle_parameter, param_config unsafe.Pointer, config_len C.int) C.int {
 	et := pserver.ElementType(param.element_type)
 	name := C.GoString(param.name)
-	content := cArrayToSlice(unsafe.Pointer(param.content), int(param.content_len))
 	pc := pserver.ParameterWithConfig{
-		Param:  pserver.Parameter{Name: name, ElementType: et, Content: content},
+		Param:  pserver.Parameter{Name: name, ElementType: et, Content: param.content, Length: para.content_len},
```

Comment:
fix done. sorry for the typo mistake, can not pass the go link problem yesterday.